PYME/recipes/pointcloud.py

```diff
@@ -251,20 +251,31 @@ class Ripleys(ModuleBase):
     inputPositions = Input('input')
     inputMask = Input('')
     outputName = Output('ripleys')
-    normalization = Enum(['K', 'L'])
+    normalization = Enum(['K', 'L', 'H'])
     nbins = Int(50)
     binSize = Float(50.)
     sampling = Float(5.)
     threaded = Bool(False)
+    dimension = Enum(['2D', '3D'])
```

Comment:
What should the default be? 

Very minor point, but is there a strong case for making this an `Enum` rather than just a `threeD = Bool()`? Using a bool would clean up a lot of the subsequent code as we ultimately just use `dimension == '3D'` to define a`three_d` boolean. From a UI standpoint, it will render as a drop down list or checkbox respectively - The ideal would likely be radio buttons, but for me I'd probably prefer the checkbox. If writing recipes by hand, it's also much easier to work out `True` and `False` than trying to figure out which string constants to use.

I shouldn't really care about this but the grammar nazi in me finds  'dimension' as a trait name a bit grating (suspect "dims", "dimensions" or "dimensionality" would be better).

Reply:
Good point. I'll change it to a bool.